src/connectors/uniswap/uniswap.ts

```diff
@@ -74,13 +74,14 @@ export class Uniswap implements Uniswapish {
     this._routerAbi = routerAbi.abi;
     this._gasLimitEstimate = UniswapConfig.config.gasLimitEstimate;
     this._router = config.uniswapV3SmartOrderRouterAddress(network);
+    this._v3Factory = config.uniswapV3FactoryAddress(network);
```

Comment:
instead of setting a variable like this, it's better to explicitly define this for each network, like the other addresses in the default uniswap template. even if they are the same, standards are good.